scripts/py_matter_yamltests/matter_yamltests/yaml_loader.py

```diff
@@ -54,6 +54,9 @@ def load(self, yaml_file: str) -> Tuple[str, Union[list, str], dict, list]:
                 config = content.get('config', {})
                 tests = content.get('tests', [])
 
+        # some defaults for global variables
+        config['LastReceivedEventNumber'] = 0
+
```

Comment:
The "default" value for the config normally live into `parser.py` in `__apply_legacy_config`. Can you move this over there ? 
`yaml_loader` is really about checking the syntax of the YAML file, not about defining values.